src/compiler/checker.ts

```diff
@@ -15459,6 +15463,39 @@ namespace ts {
                     }
                 }
             }
+
+            function getUnImplementedAncestorMembers(type: InterfaceType): SymbolTable {
+                const results: SymbolTable = {};
+
+                const baseTypes = getAbstractAncestorTypes(type);
+                for (const bt of baseTypes) {
+                    const abstractMembers = getNamedMembers(bt.symbol.members).filter(isSymbolAbstract);
+                    for (const am of abstractMembers) {

```

Comment:
You're looping over the members anyway, so just avoid the allocation of `filter`